pages/common/git-add.md

```diff
@@ -17,3 +17,7 @@
 - Also add ignored files:
 
 `git add -f`
+
+- Add parts of a file interactively
+
+`git add -p {{PATHSPEC}}`
```

Comment:
Can we use something other than "PATHSPEC"? It doesn't have to be 100% technically correct, just cover the most common cases, while being intelligible to beginners.